pages/[date]/[room]/talks/[id].js

```diff
@@ -17,66 +27,148 @@ function TalkDetails() {
     day,
     date: talkDate,
     time,
-  } = data.find((talk) => talk.id === id);
+    id: talkId,
+    isBookmarked,
+  } = talks.find((talk) => talk.id === id);
 
   const dateWithWeekday = dateFromNormalizedString(talkDate).toLocaleDateString(
     "de-DE",
     { weekday: "long", day: "numeric", month: "numeric", year: "numeric" }
   );
 
+  const firstAuthor = authors[0];
+  const coAuthors = ` ${authors.slice(1).join(", ")}`;
+
   return (
-    <StyledMain>
-      <StyledBackRoute href={`/${date}/${room}/`}>Back</StyledBackRoute>
-      <article>
-        <StyledTitle>{title}</StyledTitle>
-        <StyledAuthorList>{authors.join(", ")}</StyledAuthorList>
-        <StyledAbstract>{abstract}</StyledAbstract>
-        <InfoContainer>
-          <li>{session}</li>
-          <li>{dateWithWeekday}</li>
-          <li>{time}</li>
-        </InfoContainer>
-      </article>
-    </StyledMain>
+    <>
+      <StyledMain>
+        <StyledArticle>
+          <IconWrapper>
+            <BackButton aria-label="back button" onClick={() => router.back()}>
+              <CloseIcon />
+            </BackButton>
+            <ToggleButton
+              type={"button"}
+              onClick={() => {
+                onBookmarkToggle(talkId);
+              }}
+            >
+              {isBookmarked ? <BookmarkActive /> : <BookmarkInactive />}
+            </ToggleButton>
```

Comment:
Please add an `aria-label` here as you have done for your `BackButton`. I suggest to check all places, where you are rendering an SVG and to add an `aria-label`, if necessary.